Utilities/src/main/java/edu/stanford/hivdb/utilities/MutationFileReader.java

```diff
@@ -19,28 +19,23 @@
 package edu.stanford.hivdb.utilities;
 
 import java.io.BufferedReader;
-import java.io.FileNotFoundException;
 import java.io.IOException;
 import java.io.InputStream;
 import java.io.InputStreamReader;
 import java.util.ArrayList;
 import java.util.List;
-import java.util.regex.Matcher;
 import java.util.regex.Pattern;
-
-import org.apache.commons.lang3.StringUtils;
-import org.apache.logging.log4j.LogManager;
-import org.apache.logging.log4j.Logger;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
 
 import edu.stanford.hivdb.drugs.DrugClass;
 import edu.stanford.hivdb.mutations.Gene;
 import edu.stanford.hivdb.mutations.Mutation;
 import edu.stanford.hivdb.mutations.MutationSet;
 
 public class MutationFileReader {
-	private static final Logger LOGGER = LogManager.getLogger();
```

Comment:
It seemed like `LOGGER` was used for debugging this function so I removed since it doesn't seem necessary anymore. Please let me know if I should add it again.

Reply:
Yes we can get rid of it.